graphql/queryResolvers/getComments.test.js

```diff
@@ -0,0 +1,12 @@
+import { prisma } from '../../prisma'
+import { getComments } from './getComments'
+
+//should be added to coverageIngorePatters
```

Comment:
There are many tests like this under resolvers folder and this is a bad practice IMO. These test will never fail and we shouldn't retest third party modules. 

I've created two such pseudo-tests to pass jest coverage requirements but I'd rather add all resolvers to ignore patterns.